easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b-X11-20180604.eb

```diff
@@ -0,0 +1,1868 @@
+#
+name = 'R'
+version = '3.5.1'
+x11ver = '20180604'
+versionsuffix = '-X11-%s' % x11ver
```

Comment:
@fizwit I think we should drop this `versionsuffix`... It makes the name of the `R` module long and confusing, and we're only doing this for `R` even though `X11` is also used as a dependency in other easyconfigs.

So, let's drop the 2 lines above, and move the `'20180604'` version for `X11` into `dependencies` like we do for other easyconfigs?

Reply:
I did not refresh the PR before I pushed with style corrections.
agreed, R-3.5.1-foss-2018b.eb  is a better name.  Moved X11 to the dependencies